src/tools.cpp

```diff
@@ -638,6 +638,9 @@ MagicEffectNames magicEffectNames = {
 	{"orangeteleport",	CONST_ME_ORANGETELEPORT},
 	{"greyteleport",	CONST_ME_GREYTELEPORT},
 	{"lightblueteleport",	CONST_ME_LIGHTBLUETELEPORT},
+	{"fatal", 			CONST_ME_FATAL},
+	{"dodge", 			CONST_ME_DODGE},
+	{"hourglass",		CONST_ME_HOURGLASS},
```

Comment:
This isn't aligned though.

Reply:
I have no idea how to line this up, sorry

Reply:
it may be fine, github displays tabs in a weird way